uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/SimpleFileSystem/BlackBoxTest/SimpleFileSystemExBBTestFunction_ReadEx.c

```diff
@@ -1729,7 +1729,7 @@ BBTestReadExBasicTestCheckpoint4 (
       //
       // read
       //
-      BufferSize = ReadLength[Index];
+      BufferSize = ReadLength[Index-5];

```

Comment:
It looks like a more proper fix is to start index value for this for loop from 0  instead of 5